src/coreclr/inc/sospriv.idl

```diff
@@ -221,7 +221,6 @@ interface ISOSDacInterface : IUnknown
     // ThreadPool
     HRESULT GetThreadpoolData(struct DacpThreadpoolData *data);
     HRESULT GetWorkRequestData(CLRDATA_ADDRESS addrWorkRequest, struct DacpWorkRequestData *data);
-    HRESULT GetHillClimbingLogEntry(CLRDATA_ADDRESS addr, struct DacpHillClimbingLogEntry *data);
```

Comment:
This interface is used outside this repo. Removing methods from it would be a breaking change.

Instead, keep the method around and just change the implementation to return `E_NOTIMPL`.

Reply:
That's good to know, thanks for the follow up